pass/__init__.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+
+"""Manage passwords.
+
+This is a 'pass' wrapper extension.
+
+Synopsis:
```

Comment:
remove the docstring if it does not say more than description() and synopsis()

Reply:
Removed